rocAL/rocAL/include/pipeline/commons.h

```diff
@@ -95,7 +95,6 @@ struct Timing
     long long unsigned label_load_time= 0;
     long long unsigned bb_load_time= 0;
     long long unsigned mask_load_time = 0;
-    long long unsigned shuffle_time = 0;
```

Comment:
This has already been removed by Rajy in another PR. Not sure why it's seen in your branch. Please sync to TOT